src/components/PopupConfirmDeletion.js

```diff
@@ -0,0 +1,37 @@
+import Popup from "./Popup";
+export default class PopupConfirmDeletion extends Popup {
+  constructor({ popupSelector, handleFormSubmit }) {
+    super({ popupSelector });
+    this._handleFormSubmit = handleFormSubmit;
+    this._popup = document.querySelector(popupSelector);
```

Comment:
**No** need to search the **modal** again in the children of `Popup` because the parent has already found it in its constructor as `this._popupElement`. It’s available in any child.